libs/include/include/hpx/include/util.hpp

```diff
@@ -40,3 +41,10 @@
 #include <hpx/parallel/segmented_algorithms/traits/zip_iterator.hpp>
 #include <hpx/util/activate_counters.hpp>
 #endif
+
+#if defined(HPX_HAVE_LIB_STRING_UTIL)
+#include <hpx/string_util/case_conv.hpp>
+#include <hpx/string_util/classification.hpp>
+#include <hpx/string_util/split.hpp>
+#include <hpx/string_util/trim.hpp>
```

Comment:
```suggestion
#include <hpx/modules/string_util.hpp>
```

Reply:
Thanks I thought I did that but I may have missed it ! ^^'